installers/rpm-code/dev-build-all.sh

```diff
@@ -44,6 +44,7 @@ codes=(
 # see pyopaltools.sh
 #    pyopaltools
     opal
+    openmc
```

Comment:
Note: also update the "all" on the build machine, too, when this is released to alpha.